app/models/auction_query.rb

```diff
@@ -47,9 +45,16 @@ def complete_and_successful
       .paid
   end
 
-  def rejected
+  def completed
```

Comment:
So this is completed_and_bid_upon I guess if we want to be pedantic, but that's a mouthful